MAUI/ListView/emptyview.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: post
+title: EmptyView in .NET MAUI ListView control | Syncfusion
+description: Learn here all about EmptyView support in Syncfusion .NET MAUI ListView (SfListView) control and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# EmptyView in .NET MAUI ListView (SfListView)
+
+The `SfListView` allows to display and customize empty view content, when there is no data available to display using following properties,

```

Comment:
The `SfListView` allows to display and customize empty view content when there is no data available to display using the following properties,